play-services-core/src/main/java/org/microg/gms/auth/login/LoginActivity.java

```diff
@@ -75,6 +76,7 @@
 import static android.view.View.VISIBLE;
 import static android.view.inputmethod.InputMethodManager.SHOW_IMPLICIT;
 import static org.microg.gms.auth.AuthPrefs.isAuthVisible;
+import static org.microg.gms.checkin.CheckinPreferences.hideAppIcon;
```

Comment:
You add new imports without any need. Please remove them again

Reply:
Even though this PR made additions in 
```
play-services-core/src/main/kotlin/org/microg/gms/checkin/CheckinPreferences.kt
``` 
and 

```
play-services-base/core/src/main/kotlin/org/microg/gms/ui/Utils.kt
```